src/NServiceBus.MessagingBridge/MessageShovel.cs

```diff
@@ -10,10 +10,12 @@ sealed class MessageShovel : IMessageShovel
 {
     public MessageShovel(
         ILogger<MessageShovel> logger,
-        IEndpointRegistry targetEndpointRegistry)
+        IEndpointRegistry targetEndpointRegistry,
+        bool translateReplyToAddressForFailedMessages)
```

Comment:
Rather than the `translateReplyToAddressForFailedMessages`, should take a `FinalizedBridgeConfiguration` instance which can be injected by the container.  Will simplify DI registration of this class again as well.

Reply:
sorted